CHANGELOG.md

```diff
@@ -7,29 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.32.0] - 2022-09-12
+
 ### Changed
 
-- fix(tests): safe getter for optional inputs
-- chore: change the Dockerfile `ENTRYPOINT` definition of algorithm relying on `substratools` to pass the method to execute under the `--method-name` argument
-- tests: download tests optimized using the new substra feature: download function return the path to the downloaded file.
+- Fix(tests): safe getter for optional inputs
```

Comment:
It should be 
```diff
- Fix(tests): safe getter for optional inputs
+ fix(tests): Safe getter for optional inputs
```
as `fix` is a keyword for git conventional commit : https://www.conventionalcommits.org/en/v1.0.0/#specification

Reply:
thanks

Reply:
This spec is for commit, not for changelog though. I feel this should be in the `Fixed` part, not in the `Changed` part.